collectors/test-results/jira-xray/src/main/java/com/capitalone/dashboard/core/json/TestExecJsonParser.java

```diff
@@ -0,0 +1,16 @@
+package com.capitalone.dashboard.core.json;
+
+import com.atlassian.jira.rest.client.internal.json.JsonObjectParser;
+import com.capitalone.dashboard.api.domain.TestExecution;
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+
+/**
+ * This class will parse the JSON for a Test Execution
+ */
+// TODO: To be used in parsing TestExecution responses
+public class TestExecJsonParser implements JsonObjectParser<TestExecution> {
+    public TestExecution parse(JSONObject jsonObject) throws JSONException {
+        return null;
```

Comment:
@kumarvarun1252 is it supposed to return null

Reply:
this class is currently not being used by any clients. Added TODO